src/cpp/tests/proxyenginetest.cpp

```diff
@@ -619,7 +626,9 @@ private slots:
 	{
 		reset();
 
-		QSignalSpy spy(engine->statsManager(), SIGNAL(reported(const QList<StatsPacket> &)));
+        reportConnection = engine->statsManager()->reported.connect(
+            boost::bind(&ProxyEngineTest::appendTrackedPackets, this, boost::placeholders::_1)
+        );
```

Comment:
indentation.

also, consider moving `reportConnection` to be a local variable in the function rather than class member? that way, it disconnects at the end of the function, rather than at the next replacement